src/shared/test-utils/index.js

```diff
@@ -18,6 +19,23 @@ const localeConfig = {
             messages: idObj,
         },
     ],
+    // Consider the following React code:
+    //
+    // <Text>
+    //    <FormattedMessage id="foo" />
+    // </Text>
+    //
+    // If the "foo" key maps the message "bar", the following is rendered:
```

Comment:
Hmm, what do you think of: 

```suggestion
    // In this case, the "foo" key will resolve into the text "bar", and the following will be rendered: 
```